sql-select/inactive-customers.sql

```diff
@@ -0,0 +1,3 @@
+select *
+from "customers"
+where "isActive" = 'false'
```

Comment:
`false` is a boolean so you shouldn't put quotes around it. Postgres will convert it for you, but not all databases will.